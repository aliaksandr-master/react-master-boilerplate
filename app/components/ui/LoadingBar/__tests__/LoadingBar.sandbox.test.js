/* eslint-env browser, jest */
/* eslint-disable react/jsx-no-bind */

import React from 'react';
import renderer from 'react-test-renderer';
import { wrapSandbox } from '../../../../vendor/react-sandbox/sandbox-staging';
import sandboxLoadingBar from '../LoadingBar.sandbox';
import Test from '../../../test/Test';



describe('rendering', () => {
  wrapSandbox(sandboxLoadingBar).forEach(({ name, template, props }) => {
    it(`should render sandbox "${name}"`, () => {
      const LoadingBarSandboxComponent = template();

      const component = renderer.create(
        <Test>
          <LoadingBarSandboxComponent {...props} />
        </Test>
      );

      expect(component.toJSON()).toMatchSnapshot();
    });
  });
});
