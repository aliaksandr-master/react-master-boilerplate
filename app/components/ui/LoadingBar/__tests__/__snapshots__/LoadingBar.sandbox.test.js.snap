// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering should render sandbox "default" 1`] = `
<div>
  <div>
    <pre>
      {"style":{"position":"fixed","display":"none","top":0,"left":0,"right":0,"height":10,"zIndex":9999,"boxShadow":"0 0 3px rgba(0,0,0,0.25)"},"mode":"indeterminate"}
    </pre>
    <div />
  </div>
</div>
`;
